cmd/build_test.go

```diff
@@ -236,4 +239,96 @@ func TestBuildCmd(t *testing.T) {
 		assert.Equal(t, want, buf.String())
 		assert.NotNil(t, err)
 	})
+
+	t.Run("Output y/n message on build cmd without User-Agent", func(t *testing.T) {
+		root := newBuildCmd()
+
+		root.SetArgs([]string{"test"})
+		buf := bytes.NewBuffer(nil)
+		root.SetOut(buf)
+
+		logBuf := bytes.NewBuffer(nil)
+		logrus.SetOutput(logBuf)
+
+		configNew = func(confPath string) (config.Config, error) {
+			defaultEntry := &config.Entry{
+				Launcher: config.Launcher{
+					Version: "stable",
+					Image:   "screwdrivercd/launcher",
+				},
+				UUID: "",
+			}
+
+			return config.Config{
+				Entries: map[string]*config.Entry{
+					"default": defaultEntry,
+				},
+				Current: "default",
+			}, nil
+		}
+
+		actualOutput := captureStdout(func() {
```

Comment:
Maybe if we use `cobra.Command.Println`,  we do not need to use captureStdout. 
The buf will contain output.

Reply:
I tried but it might be difficult to me to pass with cobra.Command.Println. I'm sorry.